doc/src/yosys+odin/user_guide.rst

```diff
@@ -26,9 +28,8 @@ Synthesis Arguments
      **`--elaborator`**        **[odin (default), yosys]**       **Specify the tool that should perform the HDL elaboration**  				 	         
      **`--fflegalize`**                                          **Converts latches' sensitivity to the rising edge as required by VPR** 						 
```

Comment:
Does vpr require rising edge FFs?  @kmurray : does tatum timing analyze negative-edge triggered FFs properly (thought it did, but not sure).  If it does, we can delete the vpr part of this comment.

Reply:
This is an option I added for the Yosys+Odin-II synthesizer. When there is a falling edge latch in the BLIF file, VPR fails with the following message:
```
# Odin-II frontend
./run_vtr_flow.py test-latch.v ../arch/timing/k6_frac_N10_frac_chain_mem32K_40nm.xml

# vpr.out
Error 1: 
Type: Blif file
File: test-latch.pre-vpr.blif
Line: 14
Message: Only rising edge latches supported
```
```
# Yosys+Odin-II frontend
./run_vtr_flow.py test-latch.v ../arch/timing/k6_frac_N10_frac_chain_mem32K_40nm.xml -elaborator yosys

#vpr.out
Error 1: 
Type: Blif file
File: test-latch.pre-vpr.blif
Line: 14
Message: Only rising edge latches supported
```
```
# Yosys+Odin-II frontend with fflegalization
./run_vtr_flow.py test-latch.v ../arch/timing/k6_frac_N10_frac_chain_mem32K_40nm.xml -elaborator yosys -fflegalize
OK
```
I have attached the results of Odin-II, Yosys+Odin-II, and Yosys+Odin-II (with the fflegalize option) for a simple Verilog file below.
[Odin-II.zip](https://github.com/verilog-to-routing/vtr-verilog-to-routing/files/9415993/Odin-II.zip)
[Yosys+Odin-II.zip](https://github.com/verilog-to-routing/vtr-verilog-to-routing/files/9415995/Yosys%2BOdin-II.zip)
[Yosys+Odin-II-fflegalize.zip](https://github.com/verilog-to-routing/vtr-verilog-to-routing/files/9415994/Yosys%2BOdin-II-fflegalize.zip)